### ISSUE TITLE
We need a couple of values to be added to the analytics helper

### ISSUE THREAD

**Issue by jonathanm-tkf** [bitmovin/bitmovin-player-react-native#176]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Currently, we have the following methods from the documentation

https://developer.bitmovin.com/playback/docs/enabling-bitmovin-analytics

```js
// Update the current custom data config.
player.analyticsCollector?.setCustomDataOnce({
  customData2: 'Updated custom data field 2',
  customData4: 'Updated custom data field 4',
});
```

we need to add new methods such as 

### Expected Behavior

```js
player.analyticsCollector.setVideoId('123');
player.analyticsCollector.setTitle('Title of the Video');
player.analyticsCollector.setCdnProvider('Cdn Provider');
```

### What platform(s) are you experiencing the issue on?

- [X] Android
- [ ] Android TV / Fire TV
- [X] iOS
- [ ] tvOS

### Player React Native SDK version

0.8.0

**Comment by rolandkakonyi** [comment]
Hi @jonathanm-tkf!

`AnalyticsConfig` already offers options to assign those values on player creation.
Could this be used for your use case?

**Comment by jonathanm-tkf** [comment]
Hi @rolandkakonyi unfortunately it doesn't work for me, as we have the Next Episode functionality, which we don't destroy the player.

I'll comment in more detail:

Currently, we have the player, which has a Next Episode button, once pressed, loads the next video, for performance reasons and to not delete the current player and recreate it we use the load implementation as follows:

```js
 player.load({
        url: videoData.url,
        type: Platform.OS === 'ios' ? SourceType.HLS : SourceType.DASH,
        title: videoData.title,
        poster: videoData.posterUrl,
...
...
..
```

after this we need to do the following 

```js
      if (player.analyticsCollector) {
        player.analyticsCollector.setCustomDataOnce({
          customData2: videoData.customId,
          customData3: videoData.supplier,
        });
        player.analyticsCollector.setVideoId(videoData.id);
        player.analyticsCollector.setTitle(getTitleAnalytics(videoData.dataAnalytics));
        player.analyticsCollector.setCdnProvider(videoData.supplier);
      }
```

Do you understand what the idea is? any recommendations? or do we need to destroy the player every time we want to modify this data?

Thanks for your help

**Comment by rolandkakonyi** [comment]
Hi @jonathanm-tkf!

I see, you are correct that this is not possible right now without creating a new player instance.

You could implement this in `AnalyticsModule` yourself or this would be a feature request.

We have further plans with our analytics integration, we can get back to you during next week.

**Comment by jonathanm-tkf** [comment]
Thanks for the effort unfortunately I have too little time to contribute apologies, it is something we have today and I would not like to change the way of implementation, I look forward to the update, thanks again. If it is not implemented and I can help when I am free I will gladly do so.
Regards.

**Comment by rolandkakonyi** [comment]
@jonathanm-tkf we are already working on a solution for this, you can watch #184 for the details.

I will post it here once it is released.

**Comment by rolandkakonyi** [comment]
Hi @jonathanm-tkf, we just released [`v0.9.0`](https://github.com/bitmovin/bitmovin-player-react-native/releases/tag/v0.9.0) with support for the above use cases.

Now you can call the following API to update source-specific values for the analytics collector:
```ts
player.analyticsCollector.addSourceMetadata({
  videoId: 'new video ID',
  title: 'new video title',
  path: 'new path',
  cdnProvider: 'new CDN provider' // your new CDN provider,
});
```

Please see [`SourceMetadata`](https://github.com/bitmovin/bitmovin-player-react-native/blob/8bfe20c0f08bacf1901f3ed7ff86ffa00e3110c2/src/analytics/config.ts#L221) for all options.
